vunit/sim_if/activehdl.py

```diff
@@ -458,7 +458,8 @@ def _create_batch_script(common_file_name, load_only=False):
 
     def _create_gui_script(self, common_file_name, config):
         """
-        Create the user facing script which loads common functions and prints a help message
+        Create the user facing script which loads common functions and prints a help message. 
```

Comment:
Trailing space on this line.

Reply:
Please also add a news fragment for the release notes, see https://vunit.github.io/contributing.html#release-notes-instructions